SimulationAgent/Program.cs

```diff
@@ -12,6 +13,12 @@ public class Program
     {
         static void Main(string[] args)
         {
+            // Temporary workaround to allow twin JSON deserialization
+            JsonConvert.DefaultSettings = () => new JsonSerializerSettings
```

Comment:
do you want to keep this change long term?  i.e. should we track removing it w/ a github issue?

Reply:
created issue https://github.com/Azure/device-simulation-dotnet/issues/108